copier/copier.go

```diff
@@ -1186,11 +1186,14 @@ func copierHandlerGet(bulkWriter io.Writer, req request, pm *fileutils.PatternMa
 								return filepath.SkipDir
 							}
 							return nil
+						} else if os.IsNotExist(err) {
+							logrus.Warningf("copier: file disappeared while reading: %q", path)
+							return nil
 						}
 						return errors.Wrapf(err, "copier: get: error reading %q", path)
 					}
 					if info.Mode()&os.ModeType == os.ModeSocket {
-						logrus.Warningf("buildah/copier: skipping socket %q", info.Name())
+						logrus.Warningf("copier: skipping socket %q", info.Name())
 						return nil
```

Comment:
Removed this because the copy code is shared with podman, and no other messages include `buildah` in them.